### DISCUSSION TITLE
Deployed app error: Private functions

### DISCUSSION THREAD

**Discussion by manuelbasanta** [vercel/community#3004] [Help]
### Summary

I'm getting the following error on Safari browsers: 
Unhandled Promise Rejection: SyntaxError: Unexpected private name #getFunction. Cannot parse class method with private name.

This happens on the client and only on the deployed app, doesn't occur locally. I really do not know how to debug this, I think it has something to do with the bundle of the app, I'm using next 13 with /app directory.

My next.config:

````
const nextConfig = {
    sassOptions: {
        additionalData: '@import "styles/_variables.scss";',
    },
    reactStrictMode: true,
    webpack: config => {
        config.resolve.fallback = { fs: false, net: false, tls: false };
        return config;
    }
};
````

Any ideas on how to deal with this? I'll share the deployed app.

<img width="1262" alt="Captura de Pantalla 2023-06-26 a la(s) 09 08 22" src="https://github.com/vercel/community/assets/26985597/e5bfcb04-d41d-4a67-a001-511bceb85c7f">

### Example

https://blockchaindeals.vercel.app/

### Steps to Reproduce

Just visit the root page of the app. This error might not happen in newer versions of macos or safari. I can reproduce the error on a macbook pro 2017 running Safari 14.1.2 on macOs Big Sur

**Comment by amyegan** [comment]
Hi @manuelbasanta. It looks like you might have solved this already. I don't get the error when I visit the root page of the app now.

**Reply by manuelbasanta** [reply]
I've edited the post with more data. I can see the error more clearly now: Unhandled Promise Rejection: SyntaxError: Unexpected private name #getFunction. Cannot parse class method with private name.

**Comment by manuelbasanta** [comment]
Hi @amyegan, thanks for checking, I've checked this on an iphone 13 and it also works fine, but on an older iphone 7 and on a macbook pro 2017 running Safari 14.1.2 on macOs Big Sur and both crash like I showed. The former (a newer device/os/safari version) is probably your case as well.
